runhouse/cli_utils.py

```diff
@@ -241,7 +241,7 @@ def print_envs_info(servlet_processes: Dict[str, Dict[str, Any]], current_cluste
         # case where the default env doesn't hve any other resources, apart from the default env itself.
         console.print(f"{BULLET_UNICODE} {default_process_name}")
         console.print(
-            f"{DOUBLE_SPACE_UNICODE}This environment has only python packages installed, if provided. No "
+            f"{DOUBLE_SPACE_UNICODE}This process has only python packages installed, if provided. No "
```

Comment:
we no longer install things in a "process" so not sure if this is the best way to explain it